src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1237,6 +1238,25 @@ interface IJsonSerializerStrategy
         string MapDictionaryKeyToFieldName(string stringKey);
     }
 
+#if SIMPLE_JSON_INTERNAL

```

Comment:
Oh right, gotcha! Let's ping @prabirshrestha and see if he can shed some light on the `[GeneratedCode("simple-json", "1.0.0")]` attribute and whether or not we should add it to the new class that's embedded in the file